specification/datadog/resource-manager/readme.az.md

```diff
@@ -9,4 +9,10 @@ az:
   package-name: azure-mgmt-datadog
 az-output-folder: $(azure-cli-extension-folder)/src/datadog
 python-sdk-output-folder: "$(az-output-folder)/azext_datadog/vendored_sdks/datadog"
+
+directive:
+  - where:
+      group: marketplace-agreement
+    set:
+      group: terms
```

Comment:
The 'terms' has been used in the previous CLI version. Should it be 'term' instead of 'terms' in the CLI?